src/platform-service/values.yaml

```diff
@@ -9,8 +9,8 @@ global:
     SMTP_PASSWORD: ""
     SMTP_HOST: ""
     SMTP_USERNAME: ""
-    SMTP_SSL: ""
-    SMTP_PORT: ""
+    SMTP_SSL: "true"
+    SMTP_PORT: "465"
```

Comment:
if we do not specify here, does not components consuming smtp default to this value ?